php/test/config.php

```diff
@@ -0,0 +1,2673 @@
+            'isWs' => false,
+            'public' => false,
+            'args' => [$symbol],
+            'skip' => $is_spot ? 'only run for swap markets' : null,
+        ),
+        'fetchFundingHistory' => array(
+            'testFile' => 'fetchFundingHistory',
+            'isWs' => false,
+            'public' => false,
+            'args' => [$symbol],
+            'skip' => $is_spot ? 'only run for swap markets' : null,
+        ),
+        'watchOHLCV' => array(
+            'testFile' => 'watchOHLCV',
+            'isWs' => true,
+            'public' => true,
+            'args' => [$symbol],
+        ),
+        'watchTicker' => array(
+            'testFile' => 'watchTicker',
+            'isWs' => true,
+            'public' => true,
+            'args' => [$symbol],
+        ),
+        'watchOrderBook' => array(
+            'testFile' => 'watchOrderBook',
+            'isWs' => true,
+            'public' => true,
+            'args' => [$symbol],
+        ),
+        'watchTrades' => array(
+            'testFile' => 'watchTrades',
+            'isWs' => true,
+            'public' => true,
+            'args' => [$symbol],
+        ),
+        'watchTickers with symbol' => array(
+            'testFile' => 'watchTickers',
+            'isWs' => true,
+            'public' => true,
+            'args' => [$symbol],
+        ),
+        'watchTickers with symbol channel name' => array(
+            'testFile' => 'watchTickers',
+            'isWs' => true,
+            'public' => true,
+            'args' => [$symbol, array(
+    'name' => 'ticker',
+)],
+        ),
+        'watchTickers with symbol channel bookTicker' => array(
+            'testFile' => 'watchTickers',
+            'isWs' => true,
+            'public' => true,
+            'args' => [$symbol, array(
+    'name' => 'bookTicker',
+)],
+        ),
+        'watchBalance' => array(
+            'testFile' => 'watchBalance',
+            'isWs' => true,
+            'public' => false,
+            'args' => [$code],
+        ),
+        'watchMyTrades' => array(
+            'testFile' => 'watchMyTrades',
+            'isWs' => true,
+            'public' => false,
+            'args' => [$symbol],
+        ),
+        'watchOrders' => array(
+            'testFile' => 'watchOrders',
+            'isWs' => true,
+            'public' => false,
+            'args' => [$symbol],
+        ),
+        'watchPosition' => array(
+            'testFile' => 'watchPosition',
+            'isWs' => true,
+            'public' => false,
+            'args' => [$symbol],
+        ),
+        'watchPositions' => array(
+            'testFile' => 'watchPositions',
+            'isWs' => true,
+            'public' => false,
+            'args' => [$symbol],
+        ),
+    ),
+    'ace' => array(
+        'skip' => 'temp',
+        'skipWs' => true,
+        'loadMarkets' => array(
+            'skippedProperties' => array(
```

Comment:
I definitely think that the skippedProperties should not exist in this file, instead those skip settings can be extended from `skip-tests.json`. 
at this first stage, i think we should not mix the skipped things from this file. later in future, we can move, but at this moment, there are some specific that suggests me to have `config` tests and `skips` separated.